### ISSUE TITLE
appcenter: command not found. Mac OS and ZSH

### ISSUE THREAD

**Issue by otaviogaiao** [Microsoft/appcenter-cli#460]
I'm installing appcenter on my react native project and I'm following the instructions on the docs. However, I can't use any appcenter command on my terminal.

I installed appcenter-cli a bunch of times (tried as root and as normal user). But nothing. I always get `zsh: command not found: appcenter`. 

My setup is

Mac OS High Sierra, 10.13.5
Node 9.11.2
npm 5.6.0

Any help is appreciated.

**Comment by amchew** [comment]
Hey there @otaviogaiao! Let's debug. First, you have to install Node.js version 8 or higher. When you run `npm install -g appcenter-cli`, what is the error message that appears?

**Comment by amchew** [comment]
@otaviogaiao no problem, and cool 💯 !

**Comment by bgold0** [comment]
@amchew Hi Amanda, hoping you can help. I'm oddly getting the same issue here

```
$ npm install -g appcenter-cli
npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
/usr/local/Cellar/node/9.11.1/bin/appcenter -> /usr/local/Cellar/node/9.11.1/lib/node_modules/appcenter-cli/bin/appcenter.js
+ appcenter-cli@1.1.5
updated 1 package in 11.82s

$ appcenter
-bash: appcenter: command not found
```

Not sure what I'm doing wrong.

**Comment by jpacyfik** [comment]
I have the same problem on my new machine. 
npm: 5.6.0
node: 8.11.3

Any news?

**Comment by minhnguyenwp** [comment]
BLOCK 1
```
$ npm install -g  appcenter-cli@2.1.1
/Users/mm/.nvm/versions/node/v8.16.1/bin/appcenter -> /Users/mm/.nvm/versions/node/v8.16.1/lib/node_modules/appcenter-cli/bin/appcenter.js

> fast-xml-parser@3.12.20 postinstall /Users/mm/.nvm/versions/node/v8.16.1/lib/node_modules/appcenter-cli/node_modules/fast-xml-parser
> node tasks/postinstall.js || exit 0

Love fast-xml-parser? Check https://amitkumargupta.work for more projects and contribution.

npm WARN webpack-cli@3.3.7 requires a peer of webpack@4.x.x but none is installed. You must install peer dependencies yourself.

+ appcenter-cli@2.1.1
added 496 packages from 433 contributors in 18.045s
$ appcenter -v
```
BLOCK 2
```
$ appcenter -v
-bash: appcenter: command not found
$ nvm use --delete-prefix v10.15.3
N/A: version "v10.15.3 -> N/A" is not yet installed.

You need to run "nvm install v10.15.3" to install it before using it.
$ nvm use --delete-prefix 8
Now using node v8.16.1 (npm v6.4.1)
$ appcenter -v
appcenter command requires at least node version 10.0.0.
You are currently running version v8.16.1.
Please upgrade your version of node.js to at least 10.0.0
$ nvm use --delete-prefix 10
Now using node v10.16.3 (npm v6.9.0)
$ appcenter -v
-bash: appcenter: command not found
```

- MACOS 10.14.5
- Node 8 / 10 - npm v6.9.0
Anyone tell me what should i do now?

**Comment by minhnguyenwp** [comment]
Oh, appcenter@2.1.1 only works on node 10.0.0. not work on node v10.16.3. thanks

**Comment by MarvanN** [comment]
appcenter version 2.1.2
node v10.16.3
react-native . 0.58.5
while linking react-native link appcenter
error shows  command not found

**Comment by sergii-frost** [comment]
Is it still not working with these versions of tools?
```
appcenter-cli v2.3.0
node v10.16.3
npm v6.9.0
```

As mentioned in one of the comments before, appcenter-cli is installed under another version of node: 

```
➜  npm install -g appcenter-cli
/usr/local/Cellar/node/10.8.0/bin/appcenter -> /usr/local/Cellar/node/10.8.0/lib/node_modules/appcenter-cli/bin/appcenter.js
npm WARN webpack-cli@3.3.10 requires a peer of webpack@4.x.x but none is installed. You must install peer dependencies yourself.

+ appcenter-cli@2.3.0
updated 1 package in 16.073s
➜  node -v
v10.16.3
➜  npm -v
6.9.0
➜  appcenter
zsh: command not found: appcenter
```

PS: as a workaround I can run it via `/usr/local/Cellar/node/10.8.0/bin/appcenter ...`

**Comment by ebouJ** [comment]
@sergii-frost can you explain further on how I can run the workaround?

**Comment by ravindranpandu** [comment]
For anyone else still facing this issue, try using yarn to install appcenter-cli, i spent almost an hour and at last went with yarn and it works as a charm

`yarn global add appcenter-cli`

**Comment by SanKlein** [comment]
This fixed it. Thank you, Ravindran!

**Comment by leotm** [comment]
Getting similar, i'll try figure

![Screenshot 2021-08-10 at 18 15 51](https://user-images.githubusercontent.com/1881059/128904820-2bdb99c9-a646-4a9a-8b3e-c1a56f7200de.png)

**Comment by leotm** [comment]
Figured in the end

![Screenshot 2021-08-11 at 08 56 36](https://user-images.githubusercontent.com/1881059/128991829-133dcc53-2886-4e76-80a8-0e0d87e59460.png)

tl;dr

```sh
echo 'export PATH="/opt/homebrew/Cellar/bin:$PATH"' >> /Users/<u>/.zprofile
```

(to not restart ur terminal)

```sh
source ~/.zprofile
```

**Comment by kati-neville** [comment]
This worked for me, thanks @ravindranpandu. Just used yarn and it worked 🎉

**Comment by faizanali1192** [comment]
For anyone else still facing this issue, try this command. This worked for me!!

`npm install --location=global appcenter-cli`
